rfcs/pagination-component.md

```diff
@@ -38,7 +38,7 @@ export default () => <StatefulPagination numPages={6} />;
   Callback for prev button click
 * `onNextClick: (event) => void` - Optional
```

Comment:
Should this and `onPrevClick` go to the new API too, since they're technically custom handlers (custom name, not expected in DOM)?